CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v2.3.0 (Not released)
+
+### `@liveblocks/react-lexical`
+
+- New default components: `AnchoredThreads` and `FlatingThreads` to display
+  threads that are tied to a specific part of the document, similar to Notion,
+  Linear.
```

Comment:
@CTNicholas Would love an extra set of eyes to help make this better.

Reply:
I'll add more to this after we've figured out the docs!